SMA_FluxExtraction.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>When you print this here, suggest to add information on what exactly it is printing:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">print("major-axis: %.2f px/beam vs. fitted result: %.2f px/beam"%(beam[0]/cdelt,Fit_params[2]))

print("minor-axis: %.2f px/beam vs. fitted result: %.2f px/beam"%(beam[1]/cdelt,Fit_params[3]))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>